### ISSUE TITLE
Source of inpsiration

### ISSUE THREAD

**Issue by pums974** [smacke/ffsubsync#95]
Hi,

I made a project oddly similar to yours just a month or two before you started yours.
[srtsync](https://github.com/pums974/srtsync)

Have I been a source of inspiration ?

If so, I would be glad to be cited somewhere ...

Anyway, my project was just a PoC and is unmaintained ever since.
I'm glad you pushed it further.

**Comment by smacke** [comment]
Hi @pums974, after I released ffsubsync (originally called subsync but renamed due to a PyPI collision) I did stumble across your srtsync, but it did not originally serve as inspiration for this project. I didn't look too much into what folks had already created to be honest, otherwise I would have thought of a different name given that subsync was already taken :)

I think there may be a one other project as well that also uses FFT to align / sync subtitles but I'm having trouble digging it up. In some sense, it's a very natural idea so I'm not surprised multiple people had it independently; the timing is an interesting coincidence though.

Even though this work was done independently, I plan to eventually link to some other projects in the readme as alternatives including your srtsync, sc0ty/subsync, and kaegi/alass, since all the tools (including my own) have various not-necessarily-overlapping failure modes.
